en/docs/install-and-setup/setup/sso/configuring-external-idp-using-oidc-for-multi-tenancy.md

```
@@ -0,0 +1,358 @@
+-   Expand **Inbound Authentication Configurations** → **OAuth/OpenID Connect Configuration** → click **Configure**. Set **Callback URL** to `https://localhost:9443/commonauth`. Click **Update**.
+
+    [![Multi Tenant SP OIDC Configs]({{base_path}}/assets/img/setup-and-install/multi-tenant-sso/multi-tenant-sp-oidc-config.png)]({{base_path}}/assets/img/setup-and-install/multi-tenant-sso/multi-tenant-sp-oidc-config.png)
+
+-   Expand **Local & Outbound Authentication Configurations**. Select previously created IdP as **Federated Authenticator**. Check below configs:
+
+    [![Multi Tenant SP Outbound Configs]({{base_path}}/assets/img/setup-and-install/multi-tenant-sso/multi-tenant-sp-outbound-configs.png)]({{base_path}}/assets/img/setup-and-install/multi-tenant-sso/multi-tenant-sp-outbound-configs.png)
+
+### Step 5 - Configure Tenanted IdP
+
+-   Log in to carbon portal as tenant admin.
+-   Repeat [Step 3](#step-3-configure-federated-authenticators). But this time pick **OAuth2/OpenID Connect Configuration** instead of **Multi Tenant Authenticator Configuration**.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)